### PR TITLE
Add news post for playtest-20200426.

### PR DESCRIPTION
**Pull request by pchote**
TODO:
 - [x] Finish and tag release
 - [x] Set release ID

Screenshot 2020-04-25 at 20 51 35
Screenshot 2020-04-25 at 21 00 15

### CODE REVIEW

File: content/news/playtest-20200426.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+
+Another weekend, another playtest! To help a small number of players who are experiencing problems with poor GPU drivers, we are today releasing [playtest-20200429](/download/) which restores the ability to use OpenGL 2.1.
+
```
Comment (reaperrr) [inline content/news/playtest-20200426.md:6]: The link above says 29 instead of 26.

**Comment by reaperrr** [comment]
Otherwise 👍 after the TODOs are resolved.

File: content/news/playtest-20200426.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Support for legacy OpenGL 2.1 drivers has been restored.
+</div>
+
+This playtest also includes a scripting fix for two RA campaign missions, but no other gameplay changes. We believe that all of the important issues have now been resolved, so plan a final release in the near future if no issues are reported with the updated graphics support.
+
+</div>
+</div>
+
+For more information, see the previous news posts for [playtest-20200418](/news/playtest-20200418/), [playtest-20200329](/news/playtest-20200329/), and the [original playtest-20200303 announcement](/news/playtest-20200303/) post is included below:
```
Comment (abcdefg30) [inline content/news/playtest-20200426.md:20]: `which is included below`?

Comment (reaperrr) [reply content/news/playtest-20200426.md:20]: or just `included below` without the `is`.

**Comment by pchote** [comment]
Updated.

**Pull request merged**